eth-bytecode-db/eth-bytecode-db-server/tests/verification_test_helpers/mod.rs

```diff
@@ -242,4 +242,93 @@ pub mod test_cases {
             "Sources returned on verification and search differ"
         );
     }
+
+    pub async fn test_verify_same_source_twice<Service, Request>(
+        test_suite_name: &str,
+        service: Service,
+        route: &str,
+        verification_request: Request,
+        source_type: SourceType,
+    ) where
+        Service: VerifierService<smart_contract_verifier_v2::VerifyResponse>,
+        Request: Serialize,
+    {
+        let db = init_db(test_suite_name, "test_verify_same_source_twice").await;
```

Comment:
not sure, maybe not just "test_verify_same_source_twice" but 

`test_suite_name + "_test_verify_same_source_twice"`

just in case two tests will run at the same time